css/style.css

```diff
@@ -1,19 +1,202 @@
+/*
+/*
+/*
+/* --------- General Style --------- */
+/*
+*/
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  color: black;
+}
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  display: grid;
+  grid-template-rows: auto auto 1fr auto;
+  grid-template-columns: auto 1fr auto;
+  margin: 10px;
+}
+
+a:hover {
+  color: orange;
+}
+
+a,
+li {
+  text-decoration: none;
+  color: black;
+}
+
+button {
+  padding: 20px;
+  border: 0px;
+  background: none;
+  font-weight: 100;
+}
+
+hr {
+  display: block;
+  margin-top: 40;
+  margin-bottom: 0.5em;
+  margin-left: 100px;
+  margin-right: 100px;
+  border-width: 1px;
+  opacity: 30%;
+}
+
+/* -------  IDs --------- */
+
+#navbar-button {
+  padding: 10px;
+}
+
+/* -------- Header adn Main ------ */
+
+header {
+  grid-column: 1 / -1;
+  display: flex;
+  flex-direction: row;
+  flex-direction: row;
+  align-content: center;
+  justify-content: space-around;
+  font-weight: 100;
+  background-color: #fdf6f8;
+}
+
+.navbar-logo {
+  width: 2%;
+}
+
+.main > h1 {
+  font-weight: 100;
+  font-size: 45px;
+}
+
+/* ---------- First Section -------- */
+
+.first-section {
+  background-image: url("../img/first-background.jpg");
+  background-size: cover;
+  grid-column: 1 / -1;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: aliceblue;
+  margin-bottom: 5%;
+}
+
+.first-section > h1 {
+  font-weight: 100;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+.first-section > h3 {
+  font-weight: 100;
+  margin-top: 0px;
+}
+
+.first-section > button {
+  font-weight: 100;
+  margin-bottom: 5%;
+}
 
+/* ---------- Second Section -------- */
 
+.second-section {
+  grid-column: 1 / -1;
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+  justify-content: center;
```

Comment:
try justify-content: space-around; in addition to give a specific width I.E. 200px to each of the subsections main-links